store/filestore/message_partition.go

```diff
@@ -0,0 +1,640 @@
+package filestore
+
+import (
+	"encoding/binary"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+	"sync"
+	"time"
+
+	"github.com/smancke/guble/store"
+
+	log "github.com/Sirupsen/logrus"
+)
+
+var (
+	MAGIC_NUMBER        = []byte{42, 249, 180, 108, 82, 75, 222, 182}
+	FILE_FORMAT_VERSION = []byte{1}
+	MESSAGES_PER_FILE   = uint64(10000)
+	INDEX_ENTRY_SIZE    = 20
+)
```

Comment:
Variables could be renamed to comply with Go naming rules (no underscores, but camelcase)